index.d.ts

```diff
@@ -179,6 +179,7 @@ interface BottomSheetProps extends ViewProps {
   onDonePress?: () => void;
   valueExtractor?: () => {};
   labelExtractor?: () => {};
+  searchbarProps?: object;
```

Comment:
@Tsudhishnair LGTM. Have one suggestion. Instead of supporting just an object blindly. Can you try adding types to `searchBarProps` so that developers wouldn't have to guess the props? You can try integrating existing `SearchBarProps` maybe.